data/sql_incremental_aggregates/prepare_schedule_a_aggregate_size.sql

```diff
@@ -27,11 +27,15 @@ group by cmte_id, cycle, size
 ;
 
 -- Create indices on aggregate
-create index on ofec_sched_a_aggregate_size (cmte_id);
-create index on ofec_sched_a_aggregate_size (cycle);
-create index on ofec_sched_a_aggregate_size (size);
-create index on ofec_sched_a_aggregate_size (total);
-create index on ofec_sched_a_aggregate_size (count);
+create index on ofec_sched_a_aggregate_size_tmp (cmte_id);
+create index on ofec_sched_a_aggregate_size_tmp (cycle);
+create index on ofec_sched_a_aggregate_size_tmp (size);
+create index on ofec_sched_a_aggregate_size_tmp (total);
+create index on ofec_sched_a_aggregate_size_tmp (count);
+
+-- Remove previous aggregate and rename new aggregate
+drop table if exists ofec_sched_a_aggregate_size cascade;
```

Comment:
You're right, it will. :-(  I ran into this when I initially made the change because I didn't have the `cascade`.  This dependency does make it a bit difficult to account for but I'm not sure of a good way at the moment to improve it given the order of the other steps.